tests/cypress/support/commands.js

```diff
@@ -217,10 +217,17 @@ Cypress.Commands.add('closeSettings', () => {
         });
 });
 
-Cypress.Commands.add('changeAnnotationMode', (mode) => {
+Cypress.Commands.add('changeAnnotationMode', (mode, labelName) => {
     cy.get('.cvat-workspace-selector').click();
     cy.get('.ant-select-dropdown-menu-item').contains(mode).click();
     cy.get('.cvat-workspace-selector').should('contain.text', mode);
+    if (mode === 'Attribute annotation') {
+        // Select the necessary label in any case
+        cy.get('.attribute-annotation-sidebar-basics-editor').within(() => {
+            cy.get('.ant-select-selection').click();
+        });
+        cy.get('.ant-select-dropdown-menu-item').contains(labelName).click();
+    }
```

Comment:
Let it me a separated command. Like: ``changeLabel`` where mode can be detected automatically and corresponding elements are used to label changing. This command doesn't look like a right place to this ``if``

Reply:
Made it a separate command. But I don't think you can completely avoid this condition. Cypress will try to search for the change label field where it doesn't exist. What will result in an error